### ISSUE TITLE
Indicator appearance/disappearance timing

### ISSUE THREAD

**Issue by MonaTheMonad** [nathanielsimard/overview-navigation#28]
When the option `Show Window Selector when show Overview` option is enabled, and when Overview animations are on, the letter indicators seem to appear a little too soon and disappear a little too late. That is, when entering Overview, the indicators appear before the animation is even finished, and when exiting Overview, they don't disappear *until* the animation is over. Not as noticeable if animations are sped up (e.g. with Impatience), but just a minor observation.

**Comment by nathanielsimard** [comment]
Yep I noticed, thanks for the issue. I'm not sure how hard it would be to fix this since I don't play at all with animations. I think impatience is a good workaround for now if it's bothering you too much.

**Comment by MonaTheMonad** [comment]
Oh, it doesn't bother me. Just reporting bugs where I see them.

Are there any events for animation start/end, like `onOverview{Enter,Exit}{Begin,End}` or something?
